canopen/pdo/base.py

```diff
@@ -124,7 +124,7 @@ def stop(self):
             pdo_map.stop()
 
 
-class Maps(Mapping):
+class PdoMaps(Mapping[int, "PdoMap"]):
```

Comment:
This syntax is not compatible with py3.8, while it works with >py3.9. With respect to PR #428, are we aiming for minimum 3.8?

Reply:
I will revert and test it with py3.8 to verify there is no other surprises